xbmc/dialogs/GUIDialogProgress.h

```diff
@@ -48,6 +48,7 @@ class CGUIDialogProgress :
    \return true if the dialog is closed, false if the user cancels early.
    */
   bool Wait(int progresstime = 10);
+  bool WaitOnEvent(CEvent& event, int progresstime = 10);
```

Comment:
no default values for parameters

Reply:
There is default value for Wait() in a line above. This is the same function except one additional param. Sorry, I don't understand, why we can use defaults in Wait(), but cannot use in WaitOnEvent(). Moreover, I would like this default value to be in one place.

Reply:
> why we can use defaults in Wait()

we can't but its already happened. no new code with defaults. feel free to fix the old code if you have time. but not required for this PR